<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictView</class>
 <widget class="QWidget" name="DictView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QWebView" name="view">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Entries: 1-10 of 200</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbHome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to first entry</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../application.qrc">
         <normaloff>:/images/arrow-home.png</normaloff>:/images/arrow-home.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPrev">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to previous entry</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../application.qrc">
         <normaloff>:/images/arrow-prev.png</normaloff>:/images/arrow-prev.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+PgUp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to next entry</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../application.qrc">
         <normaloff>:/images/arrow-next.png</normaloff>:/images/arrow-next.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+PgDown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbEnd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to last entry</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../application.qrc">
         <normaloff>:/images/arrow-end.png</normaloff>:/images/arrow-end.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+End</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
